lib/phare/check/jshint.rb

```diff
@@ -25,7 +25,13 @@ def command
     protected
 
       def excluded_files
-        [] # TODO: Fetch the exclude list from .jshintignore
+        return [] unless configuration_file
+
+        configuration_file.split("\n").flat_map { |path| Dir.glob(path) }
+      end
+
+      def configuration_file
+        @configuration_file ||= File.exist?('.jshintignore') ? YAML::load(File.open('.jshintignore')) : false
```

Comment:
I think you should just split de file on `\n` like JSHint does here: https://github.com/jshint/jshint/blob/8e680e6accfac7c52933b4bc20a8b25c8382171a/src/cli.js#L200